src/metabase/query_processor/streaming.clj

```diff
@@ -19,7 +19,8 @@
 
 (defn- streaming-rff [results-writer]
   (fn [initial-metadata]
-    (let [row-count (volatile! 0)]
+    (let [row-count                (volatile! 0)
+          ordered-col-viz-settings (:ordered-col-viz-settings initial-metadata)]
```

Comment:
Is this used anywhere?

Reply:
Not yet. Will use it in the next PR to format XLSX exports. I can remove it from this PR.